src/registry.json

```diff
@@ -74,6 +74,9 @@
       "versions": {
         "1.4.2": {
           "checksum": "qSwGU4febad/0NYOJqxdO8oqmdBD8xgh8+0wOrItO8k="
+        },
```

Comment:
Should we remove the old version?

Reply:
Ah yeah, good catch. It tries to use coinType 60. Hold on.